public/js/index.js

```diff
@@ -1458,29 +1458,26 @@ $(document).ready(function () {
     });
 
     // Helpers.
-    $('.delete-button').click(function () {
-        var $this = $(this);
-        var filter = "";
-        if ($this.attr("id")) {
-          filter += "#"+$this.attr("id")
+    $('.delete-button').click(showDeletePopup);
+
+    var prev_val = "{{branch_name}}";
```

Comment:
It does not matter as we are not using real es6 template anyway but doing just replacement with javascript. `{{` `}}` syntax in files can be easily confused with golang template code (like currently vuejs code in Gitea is using `<%` `%>` to mix vue templates in golang templates but that is quite unfriendly to html parsers and code editors)